assets/dev/js/editor/utils/images-manager.js

```diff
@@ -48,6 +48,7 @@ ImagesManager = function() {
 		} );
 	};
 
+	// Ir url is exist, return it from cache, otherwise, get it from server and trigger event.
 	self.getImageUrl = function( image ) {
```

Comment:
maybe add a callback parameter, like @matipojo said?

Reply:
I am not changing the function with the current task scope.